app/src/main/java/org/fossasia/openevent/app/event/attendees/AttendeesPresenter.java

```diff
@@ -37,11 +37,6 @@ public void attach() {
     }
 
     @Override
-    public void detach() {
```

Comment:
We were detaching presenter in `onDetach()` method of fragment which is last lifecycle method of fragment. And it is called every time while switching between fragments leads to NPE for view.
We can't detach the presenter in any of the fragment's lifecycle.